.ansible-lint

```diff
@@ -3,8 +3,6 @@ exclude_paths:
   - hosts_example.yml
 skip_list:
   - risky-shell-pipe
-  - command-instead-of-shell
-  - command-instead-of-module
```

Comment:
personally command instead of module is important, but i dont feel its a worth making a command over shell a standard

Reply:
Yes, I can follow you on that one. The `command` and `shell` "modules" are not that different. Should we keep `command-instead-of-shell` on the skip list, or move it to the warn list?

Reply:
I'd prefer to only tackle `command-instead-of-module` in this PR

Reply:
Updated PR according to your preferences.